Create 'Credentials' section of Controller UG (#469)
ansible/aap-docs: downstream/modules/platform/ref-controller-credential-vault.adoc

[id="ref-controller-credential-vault"]

= Vault

Select this credential type to enable synchronization of inventory with Ansible Vault.

//image:credentials-create-vault-credential.png[Credentials- create Vault credential]

Vault credentials require the *Vault Password* and an optional *Vault Identifier* if applying multi-Vault credentialing. 

For more information on the Multi-Vault support, refer to the [Multi-Vault Credentials] section of the
_Automation Controller Administration Guide_.

You can configure {ControllerName} to ask the user for the password at launch time by selecting *Prompt on launch*. 

> In these cases, a dialog opens when the job is launched, promoting the user to enter the password and password.

Reviewer: ```suggestion
When *Prompt on launch* is selected, a dialog opens when the job is launched, promoting the user to enter the password and password.
```
